### PR TITLE
BMCP-9: Topbar Component

### PR DESCRIPTION
**Pull request by nikhilsharma0025**
## Description
Adding topbar component using material and mobile-friendly also .

#### Modification
Topbar Component

### CODE REVIEW

File: src/components/topbar/topbar.tsx [hunk 1]
```diff
@@ -1,9 +1,102 @@
...
+          </div>
+            </Hidden>
+        <Hidden only={['md', 'xl', 'lg','sm']} >
+          <div className={classes.menubutton}>
+          <Button  ref={menuAnchorRef} onClick={openMenu}>
+         <MenuIcon/>
+        </Button>
+         </div>
+        </Hidden>
+      <Menu keepMounted open={!!open} anchorEl={menuAnchorRef.current}  onClose={closeMenu} >
```
Comment (2SkeePeR2) [inline src/components/topbar/topbar.tsx:65]: !!open ?

**Review by 2SkeePeR2 — CHANGES_REQUESTED** [review]
Add a pic of the topbar before we can approve.

File: src/components/topbar/topbar.tsx (hunk 1 above)
Comment (nik-1236789) [reply src/components/topbar/topbar.tsx:65]: remove double negation.

**Review by nik-1236789 — APPROVED** [review]

**Review by 2SkeePeR2 — APPROVED** [review]

**Comment by 2SkeePeR2** [comment]
GG :fire:

**Pull request merged**